package.json

```diff
@@ -19,12 +19,12 @@
   "dependencies": {
     "antlr4": "~4.6.0",
     "bunyan": "^1.8.9",
-    "shr-models": ">=5.0.0-beta <5.1.0"
+    "shr-models": "^4.2.1"
```

Comment:
Looks like you accidentally overwrote the new value with an older one?  Probably a result of the merge?  Should be: `"shr-models": ">=5.0.0-beta <5.1.0"`